### PR TITLE
Initialize mapping with  bytes image

### PR DESCRIPTION
**Pull request by garbervetsky**
Addresses #142 

Basic support to initialization of mappings with image of bytes. Need to add some test.

### CODE REVIEW

File: Sources/SolToBoogie/GhostVarAndAxiomGenerator.cs [hunk 1]
```diff
@@ -364,9 +397,52 @@ private BoogieAxiom GenerateAbiEncodePackedAxiomTwoArgs()
...
+            var qExpr = new BoogieQuantifiedExpr(true, new List<BoogieIdentifierExpr>() { qVar1, qVar2 }, new List<BoogieType>() { BoogieType.Ref, BoogieType.Ref}, bodyExpr, triggers);
+
             return new BoogieAxiom(qExpr);
         }
 
+        private BoogieAxiom GenerateAbiEncodePackedAxiomTwoArgsOneRef()
+        {
+            var qVar11 = QVarGenerator.NewQVar(0, 0);
+            var qVar12 = QVarGenerator.NewQVar(0, 1);
+            var qVar21 = QVarGenerator.NewQVar(1, 0);
```
Comment (shuvendu-lahiri) [inline Sources/SolToBoogie/GhostVarAndAxiomGenerator.cs]: did you already branch out of your other PR?

File: Sources/SolToBoogie/ProcedureTranslator.cs [hunk 2]
```diff
@@ -1632,6 +1639,9 @@ private void TranslateAbiEncodedFuncCall(FunctionCall funcCall, BoogieExpr lhs)
             }
             var boogieExprs = arguments.ConvertAll(x => TranslateExpr(x));
             var funcName = $"abiEncodePacked{arguments.Count}";
+            // hack
```
Comment (shuvendu-lahiri) [inline Sources/SolToBoogie/ProcedureTranslator.cs]: Oooh! What is this for?

File: Sources/SolToBoogie/GhostVarAndAxiomGenerator.cs (hunk 1 above)
Comment (garbervetsky) [reply Sources/SolToBoogie/GhostVarAndAxiomGenerator.cs]: No, the problem was that the ABIEncode additions were in my main branch, so they appeared in all the branches I created afterwards 🤦‍♂

File: Sources/SolToBoogie/ProcedureTranslator.cs (hunk 2 above)
Comment (garbervetsky) [reply Sources/SolToBoogie/ProcedureTranslator.cs]: To determine the function name according to the type of the first argument. Not supper crafted design. This is why I said "hack". I need to have this encodings for the Zeppelin case study.

Comment (garbervetsky) [reply Sources/SolToBoogie/ProcedureTranslator.cs]: I have moved this commit to another PR

**Comment by shuvendu-lahiri** [comment]
All ready to merge this PR with a test case.

**Comment by garbervetsky** [comment]
> All ready to merge this PR with a test case.

Done.

**Pull request merged**